### DISCUSSION TITLE
Only listens to a single network segment.

### DISCUSSION THREAD

**Discussion by rems25000** [telekom-security/tpotce#1244] [Q&A]
Hello,
T-pot runs on a VM on an address 192.168.10.XXX/32
If I do an open port test from this interface, port 22 and the other ports respond well.

If I go from my PC to access the interface from an IP source at 192.168.50.XXX/32

--> No port responds to me.
Where should I configure, please?

PS: I do have a functional rule between vlan 50 and 10.

Thanks

**Comment by Ganesh-403** [comment]
@rems25000 
If T-Pot is reachable from hosts in **192.168.10.0/24** but **none of the honeypot or management ports respond from 192.168.50.0/24**, even though the inter-VLAN firewall rules are correct, then the issue is most likely **host-side filtering or routing**, not the honeypots themselves.

A few things to verify:

### 1. Make sure you're testing the correct ports

Remember that T-Pot changes some default management ports:

- SSH → **64295**
- Web UI → **64297**

The honeypot ports (e.g. 22 for Cowrie) are separate from the management ports. :contentReference[oaicite:0]{index=0}

For example:

```bash
ssh -p 64295 <user>@192.168.10.xxx
```

---

### 2. Verify that T-Pot is listening on the expected interfaces

On the T-Pot host, run:

```bash
ss -tulnp
```

or

```bash
docker ps
```

Some management services are intentionally bound to localhost or protected by the T-Pot firewall, while honeypot services are exposed on the host interface.

---

### 3. Check the host firewall

T-Pot manages its own firewall configuration rather than relying on UFW. Even if your inter-VLAN firewall allows the traffic, the T-Pot host may still reject it.

Inspect the active rules:

```bash
sudo nft list ruleset
```

or, on older installations:

```bash
sudo iptables -L -n -v
```

Look for any rules that only permit traffic from the local subnet or explicitly drop traffic arriving from `192.168.50.0/24`.

---

### 4. Verify routing

From the T-Pot host:

```bash
ip route
```

And test connectivity back to your workstation:

```bash
ping 192.168.50.xxx
traceroute 192.168.50.xxx
```

Since this is inter-VLAN communication, return routing is just as important as the firewall rules.

---

### 5. Consider the recommended deployment

The T-Pot documentation recommends placing the honeypot behind a firewall, forwarding honeypot ports (typically `1–64000`) to T-Pot, while restricting management ports (`>64000`) to trusted IP addresses. This is the deployment model the project is designed around. :contentReference[oaicite:1]{index=1}

---

A few questions that would help narrow this down:

- Are **all ports** unreachable from `192.168.50.0/24`, or only the management ports?
- Does `ping` to the T-Pot host work from VLAN 50?
- What is the output of:

```bash
ip route
sudo nft list ruleset
docker ps
```

That should help determine whether the issue is with host firewall rules, Docker networking, or the inter-VLAN routing.

If this solves your problem, feel free to mark it as the accepted answer so others can find it easily.
